README.md

```diff
@@ -2,6 +2,19 @@
 
 Build images from the commandline in a convenient way.
 
+## Run via container
+
+TODO: build container automatically via GH action and put into ghcr.io
```

Comment:
Leftover TODO

Reply:
I've addressed this in a separate commit.